app/uk/gov/hmrc/nisp/views/includes/nirecordtaxyear.scala.html

```diff
@@ -19,7 +19,9 @@
 @import uk.gov.hmrc.nisp.controllers.routes
 @import uk.gov.hmrc.play.views.formatting.Dates
 @import uk.gov.hmrc.nisp.models.NpsDate
-@(niRecordTaxYear: NIRecordTaxYear)
+@import java.time._
+
+@(niRecordTaxYear: NIRecordTaxYear, timePoint: LocalDate = LocalDate.now())
```

Comment:
It is used to get the current date, for some reason I was not able to get the current date any other way.